common_test.go

```diff
@@ -4,22 +4,54 @@ import (
 	"fmt"
 	"io/ioutil"
 	"os"
+	"os/exec"
 	"path/filepath"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/suite"
 )
 
-type SimpleLinguistTestSuite struct {
+const lingustURL = "https://github.com/github/linguist.git"
+
+type EnryTestSuite struct {
 	suite.Suite
+	tmpLinguist string
```

Comment:
According to the use we are giving to this variable, I would call it localLinguistRepo or something similar.